docs/openapi.yml

```diff
@@ -513,11 +509,7 @@ components:
       example:
         runId: 870492da-ecfb-4be0-91b9-9a89ddd3db90
         runState: COMPLETED
-        runArgs:
-          email: data@wework.com
-          emailOnFailure: false
-          emailOnRetry: true
-          retries: 1
+        runArgs: "{\\\"email\\\": \\\"data@wework.com\\\", \\\"emailOnFailure\\\": false, \\\"emailOnRetry\\\": true, \\\"retries\\\": 1}"
```

Comment:
I'm not a fan of escaped json - but for Q4 in the interest of expediency, I'm okay with it.